_posts/2015-04-29-caching-external-apis.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout:       post
-title:        Caching External API's in Rails for a Ginormous Speed Boost
+title:        Caching External APIs in Rails for a Ginormous Speed Boost
```

Comment:
lets start a war. start a nuclear war: http://english.stackexchange.com/questions/503/what-is-the-correct-way-to-pluralize-an-acronym